.gitignore

```diff
@@ -5,7 +5,6 @@
 *~
 .bundle
 db/*.sqlite3
-gemfiles/*.lock
```

Comment:
I was seeing failures on my local system when running the specs that weren't happening on GitHub. I tracked this down to different versions of gems being installed locally versus CI so it makes sense to commit the lockfiles in order to achieve parity between local and CI.